packages/dht/src/connection/ConnectivityChecker.ts

```diff
@@ -35,15 +39,19 @@ export class ConnectivityChecker {
         let outgoingConnection: IConnection
         try {
             outgoingConnection = await this.connectAsync({
-                host: entryPoint.websocket?.ip, port: entryPoint.websocket?.port, timeoutMs: 1000,
+                wsServerInfo: {
+                    host: entryPoint.websocket!.host, 
+                    port: entryPoint.websocket!.port,
+                    tls: entryPoint.websocket!.tls,
+                },
                 mode: ConnectionMode.REQUEST
             })
         } catch (e) {
-            logger.error('Failed to connect to the entrypoints')
+            logger.error('Failed to connect to the entrypoints: ' + e)
```

Comment:
Why this logs and throws? Could the caller log the information if that is an error in?

(Also is it failing to connect to one entry point or multiple entry points here?)

```suggestion
            logger.error('Failed to connect to the entrypoint', { entryPoint, error: e })
```